tests/test_docker.py

```diff
@@ -172,6 +173,40 @@ def post(req, resp):
 
     event_test(agent, 'docker/instance_activate', post_func=post)
 
+def test_multiple_nics_pick_mac():
+    instance = {    "nics": [
+                        {
+                            "macAddress": "02:03:04:05:06:07",
+                            "deviceNumber": 0,
+                            "ipAddresses": [
```

Comment:
The ipAddresses property isn't needed for the function under test right? Want to remove it so that this dict can be more concise?